Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/BocListControlObjectTest.cs

```diff
@@ -373,6 +373,34 @@ public void ScreenshotTest_DerivedTypeGeneric ()
       Assert.That(() => derivedDropDown.Open(), Throws.Nothing);
     }
 
+    [Category("Screenshot")]
+    [Test]
+    public void ScreenshotTest_ValidationMarkerAndMessages ()
```

Comment:
I tried adding a test for the grid as well but I still have problems to get the screenshots to run on the TC agents. The problem with the grid is that it is not fully visible since it is too wide. This caused the screenshotting to fail since the element was not fully visible. I adjusted the test to crop a fixed area around the cell to prevent visibility issues but the tests now fail with a wonderful "A generic error occurred in GDI+." error. This error can essentially mean anything. The strange thing is that the test is exactly the same as the grid test but here it fails with the exception.

So either somebody takes a look at what might cause the GDI+ error or we scrap the tests for the grid